shell/platform/embedder/embedder.cc

```diff
@@ -46,18 +48,22 @@ const int32_t kFlutterSemanticsNodeIdBatchEnd = -1;
 const int32_t kFlutterSemanticsCustomActionIdBatchEnd = -1;
 
 static FlutterEngineResult LogEmbedderError(FlutterEngineResult code,
-                                            const char* name,
+                                            const char* reason,
+                                            const char* code_name,
                                             const char* function,
                                             const char* file,
                                             int line) {
-  FML_LOG(ERROR) << "Returning error '" << name << "' (" << code
-                 << ") from Flutter Embedder API call to '" << function
-                 << "'. Origin: " << file << ":" << line;
+  const auto file_base = (::strrchr(file, '/') ? strrchr(file, '/') + 1 : file);
+  char error[256] = {};
+  snprintf(error, (sizeof(error) / sizeof(char)) - 1,
```

Comment:
Use `sizeof(error)` here

Reply:
I wanted to keep space for the null terminator.

Reply:
Nevermind. The man page says this is unnecessary.